manifests/package.pp

```diff
@@ -106,4 +112,16 @@
     require => Package[$container_package],
   }
 
+  if $experimental_filters == true {
+    package { $repose::params::experimental_filters_packages:
+      ensure => $package_ensure, 
+      require => Package[$container_package],
+    }
+  } else {
+    package { $repose::params::experimental_filters_packages:
+      ensure => absent, 
```

Comment:
My concern here is just leaving this up to pacakge_ensure leaves the loop that some one can set experimental_filters to false and it wouldn't get removed. 
That could be the crux we have to bare to maintain backward compat with systems that might have installed experimental filters manually before this version.